python/cac_tripplanner/destinations/views.py

```diff
@@ -45,7 +45,11 @@ def isochrone(self, lat, lng, mode, date, time, max_travel_time, max_walk_distan
         isochrone_response = requests.get(self.isochrone_url, params=payload)
 
         # Parse and traverse JSON from OTP so that we return only geometries
-        json_poly = json.loads(isochrone_response.content)[0]['geometry']['geometries']
+        try:
+            json_poly = json.loads(isochrone_response.content)[0]['geometry']['geometries']
```

Comment:
Requests has a nice helper method you can use to save the call to json.loads, `isochrone_response.json()`